Fix broken link and change language block type

peterblazejewicz opened the pull request:
This commit changes:
- broken API documentation link to point to current Emotion API docs
- block code language type to `html`, so it's better rendered in documentation

Thanks!

azurecla commented:
Hi __@felixrieseberg__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (feriese). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

deneha commented:
@felixrieseberg Thanks for the contribution!  We're reviewing your request.

tamram commented:
Hey Felix -

We can't publish your article to azure.com because we can't promote tools on our official storage documentation site that the storage team does not support themselves.

For this reason, we need to close your PR.

Tamra

peterblazejewicz committed:
Fix broken link and change language block type

This commit changes:
- broken API documentation link to point to current Emotion API docs
- block code language type to `html', so it's better rendered in documentation

Thanks!
